handlers/client.go

```diff
@@ -25,11 +26,38 @@ type MistClient struct {
 	configMu        sync.Mutex
 }
 
+func (mc *MistClient) RemoveAllStreams() error {
+	c := RemoveAllStreamsCommand{Streams: map[string]Stream{}}
+	response, err := mc.sendCommand(c)
+	fmt.Printf("RemoveAllStreams %s\n", response)
+	return err
+}
+
+func (mc *MistClient) DeleteAllTriggers() error {
```

Comment:
What is the use of this function when we already have `DeleteAllTriggersFor()` deleting all triggers for the given stream?

Reply:
Removing..
Actually there is a bug when removing last trigger, as json field is defined as `omitempty`.
Was used when we were adding triggers from API server. 
Mist good practice is to define triggers via config json.
`AddTrigger()` and `DeleteTrigger()` are also planned for removal.